spec/support/mailchimp_helper.rb

```diff
@@ -0,0 +1,66 @@
+module MailchimpHelper
+  # Helper function to create a Gibbon request
+  def mailchimp_create_request
+    Gibbon::Request.new(
+      api_key: ENV['MAILCHIMP_API_KEY']
```

Comment:
Can we hardcode a fake api key in here please? We shouldn't need the real api key in the test environment.

Reply:
Ok, my intention here was never to use the real API key. I used a test list while developing these changes. The associated test uses VCR, but it actually hits the API in the absence of a VCR cassette. This requires an account/API key, list ID to be present and for those environment variables to be set in the `application.yml`. I'm certainly open to alternatives here. Wouldn't the Mailchimp account need to be accessible and manageable by a collaborator or maintainer?

Reply:
Ah yes I see and the intention @BideoWego 😄 

From my understanding, only the financial backers of the project will have access to Mailchimp. Making setting up a Mailchimp API key irrelevant and unnecessary for the average maintainer/collaborator. But with the current implementation a maintainer/collaborator will have to create one to get the tests to pass.

With this in mind, I think to test this feature we can just stub out the call to your service object in the registrations controller.